controllers/cloud.redhat.com/makers/maker.go

```diff
@@ -641,7 +670,7 @@ func processAppEndpoints(appMap map[string]crd.ClowdApp, depList []string, depCo
 		// If app has public endpoint, add it to app config
 
 		for _, deployment := range depApp.Spec.Deployments {
-			if deployment.Web {
+			if deployment.Web == true || deployment.WebServices.Public.Enabled == true {
```

Comment:
redundant `== true`?

Reply:
Yes - there was a reason - but it's gone now